docs/UserGuide.md

```diff
@@ -100,6 +100,14 @@ Format: `add n\NAME p\PHONE_NUMBER e\EMAIL a\ADDRESS fi\FINANCIALINFO s\SOCIALME
 A tag can be added with or without a value. For example, `t\client` and `t\insurance:No` are both valid tags.
 </box>
 
+<box type="warning" seamless>
+**Tag constraints:**<br>
+* Only alphanumeric characters (alphabets and numbers) are supported in tags<br>
+* Multiple of the same tags are not supported for a single contact<br>
+* Negative numbers are not supported in tag names<br>
+* Operations such as `+, -, *, /` are also not supported<br>
```

Comment:
Perhaps you can specify where they are not supported. For example: "Operations such as ... are also not supported when assigning tag values".